.env

```
@@ -42,7 +42,13 @@ NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
 
 
 NEXTAUTHURL = "http://localhost:3000"
+<<<<<<< HEAD
+NEXTAUTH_SECRET="FDZaz8bV6y8V9KTiJqjTxjs5Npvvm39hi+qmQZqK8fw=" 
+
+NEXT_PUBLIC_BASE_API="https://flower.elevateegy.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)